node/src/components/consensus/highway_core/highway_testing.rs

```diff
@@ -1228,3 +1233,31 @@ mod test_harness {
         );
     }
 }
+
+lazy_static! {
+    pub static ref ALICE_SECRET_KEY: SecretKey =
+        SecretKey::ed25519_from_bytes(&[0; SecretKey::ED25519_LENGTH]).unwrap();
+    pub static ref ALICE_PUBLIC_KEY: PublicKey = PublicKey::from(ALICE_SECRET_KEY.deref());
+    pub static ref BOB_PRIVATE_KEY: SecretKey =
+        SecretKey::ed25519_from_bytes(&[1; SecretKey::ED25519_LENGTH]).unwrap();
+    pub static ref BOB_PUBLIC_KEY: PublicKey = PublicKey::from(BOB_PRIVATE_KEY.deref());
+}
+
+/// Loads the local chainspec and overrides timestamp and genesis account with the given stakes.
+pub fn new_test_chainspec(stakes: Vec<(PublicKey, u64)>) -> Chainspec {
+    let mut chainspec = Chainspec::from_resources("local/chainspec.toml");
+    chainspec.genesis.accounts = stakes
+        .into_iter()
+        .map(|(pk, stake)| {
+            let motes = Motes::new(stake.into());
+            GenesisAccount::new(pk.into(), pk.to_account_hash(), motes, motes)
+        })
+        .collect();
+    chainspec.genesis.timestamp = Timestamp::now();
+    chainspec.genesis.highway_config.genesis_era_start_timestamp = chainspec.genesis.timestamp;
+
+    // Every era has exactly three blocks.
+    chainspec.genesis.highway_config.minimum_era_height = 2;
+    chainspec.genesis.highway_config.era_duration = 0.into();
+    chainspec
+}
```

Comment:
I'd rather put all of this into `components/consensus/test`… maybe `/utils.rs`?
It would be nice to keep concrete cryptography out of `highway_core`, which is generic in all of that.

Reply:
We can put this in `utils.rs`, sure.